chapter_12_pracic/task1.cpp

```diff
@@ -0,0 +1,64 @@
+﻿// Start with the Distance class from the ENGLCON example in Chapter 6, “Objectsand
+// Classes.” Using a loop similar to that in the DISKFUN example in this chapter, 
+// get a number of Distance values from the user, and write them to a disk file.Append them to
+// existing values in the file, if any.When the user signals that no more values will be
+// input, read the fileand display all the values.
+
+#include <fstream>  
+#include <iostream>
+
+class Distance
+{
+private:
```

Comment:
this `private` is redundant, as by default all members of class are private

Reply:
Fixed in https://github.com/Dmitriy-Romanchuk/master/pull/12/commits/fbd708b5a089fe13ce20ba7ba105e50cc1ee4e9c